backend/WebUI/api_webui.go

```diff
@@ -401,7 +408,17 @@ func JWT(email, userId, tenantId string) string {
 	claims["email"] = email
 	claims["tenantId"] = tenantId
 
-	tokenString, err := token.SignedString([]byte(os.Getenv("SIGNINGKEY")))
+	mu.Lock()
+
+	if jwtKey == "" {
+		rand.Seed(time.Now().UnixNano())
```

Comment:
You are using `math/rand` here, which is not cryptographically secure (especially when seeded with the current time). Since this is supposed to be a secret, it should not be guessable. I recommend to use [`crypto/rand`](https://pkg.go.dev/crypto/rand) here.